streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -637,8 +638,12 @@ private boolean assignTasksToClients(final Cluster fullMetadata,
                                                                    statefulTasks,
                                                                    assignmentConfigs);
 
-        log.info("Assigned tasks {} including stateful {} to clients as: \n{}.",
-                allTasks, statefulTasks, clientStates.entrySet().stream()
+        log.info("{} assigned tasks {} including stateful {} to clients as: \n{}.",
```

Comment:
nit: Assigned {} tasks including ... to {} clients as ..

Reply:
Ah, acked.